pyo3-macros-backend/src/pyclass.rs

```diff
@@ -2023,6 +2030,45 @@ fn pyclass_hash(
     }
 }
 
+fn pyclass_class_geitem(
```

Comment:
@davidhewitt do you thinnk this is a step towards the right direction? I was in doubt where is the easiest place to inject a `@classmethod` in the macros

Reply:
[mejrs](https://github.com/mejrs) feedback fixed my issue